client/src/components/Form.js

```diff
@@ -0,0 +1,88 @@
+import styled from "@emotion/styled";
+import React from "react";
+import { useForm } from "react-hook-form";
+
+export default function Form() {
+  const { register, handleSubmit, errors } = useForm();
+  const onSubmit = (data) => console.log(data);
+  console.log(errors);
```

Comment:
If it already works, do not forget to remove the console.log

Reply:
Will do this in a later step!